cosmic-applet-power/src/lib.rs

```diff
@@ -440,8 +440,8 @@ async fn lock() -> zbus::Result<()> {
             .path(session_path)?
             .build()
             .await?;
-        if session.type_().await? != SessionType::TTY {
-            session.lock().await?;
+        if session.class().await == Ok(SessionClass::User) && session.type_().await? != SessionType::TTY {
+            session.lock().await?
```

Comment:
Thanks, it might also be better to not return right away on the first error, and instead return the error after iterating through each session.

Reply:
Yeah that's also what I was considering. I think it makes more sense. Do we want to return an error if any of the sessions fail, or if all sessions fail?

I think logically, if any sessions successfully locks, it's an `Ok`?

Reply:
Ya, that sounds fine :+1:

Reply:
Updated this PR 👍